wp-content/themes/goonj-crm/functions.php

```diff
@@ -39,6 +39,16 @@ function goonj_theme_setup() {
 	add_editor_style('style.css');
 }
 
+function enqueue_custom_reset_script() {
+	wp_enqueue_script(
+		'reset-button-handler-script',
```

Comment:
Don't we already have a JS file being loaded on the page. We don't want to add a new JS everytime we write a new feature

Reply:
Oh, we already have a `main.js` file loaded on the page. Instead of adding new JavaScript files for each feature, it would be better to integrate this code into the existing `main.js` file. I'll update it accordingly